templates/docs/examples/patterns/chip/with-dismiss.html

```diff
@@ -7,8 +7,178 @@
+  <span class="p-chip__value">Bob</span>
+  <span class="p-chip__badge">0</span>
+</div>
+
+<div class="p-chip--positive">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+  <button class="p-chip__dismiss">
+    <i class="p-icon--close">Dismiss</i>
+  </button>
+</div>
+
+<div class="p-chip--positive">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+</div>
+
+<br /> 
+
+<div class="p-chip--caution">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+  <span class="p-chip__badge">0</span>
+  <button class="p-chip__dismiss">
+    <i class="p-icon--close">Dismiss</i>
+  </button>
+</div>
+
+<div class="p-chip--caution">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+  <span class="p-chip__badge">0</span>
+</div>
+
+<div class="p-chip--caution">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+  <button class="p-chip__dismiss">
+    <i class="p-icon--close">Dismiss</i>
+  </button>
+</div>
+
+<div class="p-chip--caution">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+</div>
+
+<br />
+
+<div class="p-chip--negative">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+  <span class="p-chip__badge">0</span>
+  <button class="p-chip__dismiss">
+    <i class="p-icon--close">Dismiss</i>
+  </button>
+</div>
+
+<div class="p-chip--negative">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+  <span class="p-chip__badge">0</span>
+</div>
+
+<div class="p-chip--negative">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+  <button class="p-chip__dismiss">
+    <i class="p-icon--close">Dismiss</i>
+  </button>
+</div>
+
+<div class="p-chip--negative">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+</div>
+
+<br />
+
+<div class="p-chip--negative">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+  <span class="p-chip__badge--negative">0</span>
+  <button class="p-chip__dismiss">
+    <i class="p-icon--close">Dismiss</i>
+  </button>
+</div>
+
+<div class="p-chip--negative">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+  <span class="p-chip__badge--negative">0</span>
+</div>
+
+<div class="p-chip--negative">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+  <button class="p-chip__dismiss">
+    <i class="p-icon--close">Dismiss</i>
+  </button>
+</div>
+
+<div class="p-chip--negative">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+</div>
+
+<br />
+
+<div class="p-chip--information">
+  <span class="p-chip__lead">Owner</span>
+  <span class="p-chip__value">Bob</span>
+  <span class="p-chip__badge">0</span>
```

Comment:
Are we adding badges at the same time?
Badges were supposed to be a separate component you can place anywhere, not just part of a chip, having `p-chip__badge` makes it child of this component, not reusable elsewhere.

I think we should wait with adding badges until we add them as independent component everywhere.

Reply:
I'm just working on top of @sowasred2012 's pr, not sure if he intended to do chips as part of this?

Reply:
We have badges planned separately, so I would suggest dropping them from examples and styles for now.

Reply:
Yeah drop anything you don't like from my work - I was following the designs, but it was all WIP.

Reply:
fixed